### PR TITLE
[No Reviewer] Stats api update

### PR DESCRIPTION
**Pull request by johadalin**
Just altering tags from vector to map to fit with new Chronos Api.

### CODE REVIEW

File: src/session_manager.cpp [hunk 1]
```diff
@@ -390,7 +390,8 @@ void SessionManager::send_chronos_update(std::string& timer_id,
                                          const std::string& opaque_data,
                                          SAS::TrailId trail)
 {
-  std::vector<std::string> tags = {"CALL"};
+  std::map<std::string, uint32_t> tags; tags["CALL"] = 1;
+
```
Comment (chris-elford-metaswitch) [inline src/session_manager.cpp]: Extra blank line?

File: src/session_manager.cpp [hunk 2]
```diff
@@ -390,7 +390,8 @@ void SessionManager::send_chronos_update(std::string& timer_id,
                                          const std::string& opaque_data,
                                          SAS::TrailId trail)
 {
-  std::vector<std::string> tags = {"CALL"};
+  std::map<std::string, uint32_t> tags; tags["CALL"] = 1;
```
Comment (chris-elford-metaswitch) [inline src/session_manager.cpp]: [The online C++ reference](http://en.cppreference.com/w/cpp/container/map/map) gives some examples of creating maps with different constructors.

You should consider using the technique in example 5.

**Comment by chris-elford-metaswitch** [comment]
Not much to add here - this is good to go once you've addressed the minor markups.

File: src/session_manager.cpp (hunk 2 above)
Comment (johadalin) [inline src/session_manager.cpp]: Thanks for finding that. not sure how i missed those examples.

**Pull request merged**